test/bump/cli_test.rb

```diff
@@ -1,7 +1,32 @@
 require "test_helper"
 
-class Bump::CLITest < Minitest::Test
-  def test_that_it_has_a_version_number
+describe Bump::CLI do
+  before do
+    @original_path = ENV['PATH']
```

Comment:
Oh yeah probably better 👍 

As a side note, actually I'm not that fan but this whole thing, as I wrote `feature` specs for the CLI and wanted to have the executable in the path of specs context (not fan because of https://github.com/bump-sh/bump-cli/pull/1/files#diff-ec2671a5dace290a032b5858f0b7daf2R6).
It's much simpler than going with things like https://github.com/cucumber/aruba, but I think testing with instantiating the objects (as we do in the executable) could be simpler.

Reply:
As you wish, I don't have any strong opinion on this.